miri

```diff
@@ -211,6 +216,9 @@ fmt)
     find "$MIRIDIR" -not \( -name target -prune \) -name '*.rs' \
         | xargs rustfmt +$TOOLCHAIN --edition=2021 --config-path "$MIRIDIR/rustfmt.toml" "$@"
     ;;
+rustup)
```

Comment:
This should be up there with the early commands, not down here where it gets a bunch of env vars it does not want.